src/components/shared/layout/Filterbar.vue

```diff
@@ -1,12 +1,14 @@
 <template>
     <div class="ui secondary basic segment">
         <div class="ui text menu">
-            <div class="item">
```

Comment:
Were these changes requested? When we built this filterbar, we were told it was to promote the consistency of having a search bar on the left and the other options on the right.